source/mod_ffield_aux.f90

```diff
@@ -820,13 +820,13 @@ subroutine TLie2(this,x,px,y,py,zb,deltap1)
 
     ! Check size vectors xpow and ypow   (Prevent SIGFPE)
     ! ---------------------------------------------------------------------------------------------- !
-    log_tmp=abs(log10_mb(abs(x)))
+    log_tmp=log10_mb(abs(x))
     if (log_tmp*(this%n)>230) then
```

Comment:
I don't know the logics behind (hence maybe my question is very naive), but since you have removed the `abs` on `log_tmp`, are you sure that the if statement still makes sense as is?

Reply:
This part was to prevent the signal SIGFPE. n and m are the maximum exponent for respectively x and y in the horner routine and can have the value 30. This prevent an overflow but adding the abs outside the log generate an error for on axis (x or y) particles.
If the SIGFPE signal is not taken into account for the release anymore then this section can be removed.

Reply:
As a side not, ffindex(:) is equal to 0 most of the time, positive if the code use the fringe field procedure and negative if we want to skip a multipole.

Reply:
Hi @tpugnat ,

I think that you still get a `SIGFPE` when `x=0` or `y=0` (on-axis particles) - from `log10_mb(zero)`.

Moreover, the left-hand part of the check under `if` statement was always positive before this bug fix, whereas now it can be either positive or negative, and it is not clear to me that comparing the left-hand side (positive or negative) with a number (always) positive is what you want to do...

I don't have any other objection - in the end, you are the main developer and user of this part

Reply:
Since subnormal numbers are handled anyway, can you rewrite this to drop the log10 call in the if-statement and just check against a large number? The log function is relatively slow.

Reply:
I will make the change.